helm-chart/templates/secrets.yaml

```diff
@@ -0,0 +1,78 @@
+---
+{{- if .Values.secrets.localSettings }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: local-settings-{{ .Values.deployEnv }}
+  labels:
+    deployEnv: "{{ .Values.deployEnv }}"
+type: Opaque
+data:
+  local_settings.py: {{ .Values.secrets.localSettings | b64enc }}
+{{- end }}
+---
+{{- if .Values.web.secrets.googleClient }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: google-client-secret-{{ .Values.deployEnv }}
+  labels:
+    deployEnv: "{{ .Values.deployEnv }}"
+type: Opaque
+data:
+  client_secrets.json: {{ .Values.web.secrets.googleClient | b64enc }}
+{{- end }}
+---
+{{- if .Values.varnish.secrets.varnish }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: varnish-secret-{{ .Values.deployEnv }}
+  labels:
+    deployEnv: "{{ .Values.deployEnv }}"
+type: Opaque
+data:
+  varnish-secret: {{ .Values.varnish.secrets.varnish | b64enc }}
+{{- end }}
+---
+{{- if .Values.web.secrets.schoolLookup }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: school-lookup-data-{{ .Values.deployEnv }}
+  labels:
+    deployEnv: "{{ .Values.deployEnv }}"
+type: Opaque
+data:
+  schools.tsv: {{ .Values.web.secrets.schoolLookup | b64enc }}
+{{- end }}
+---
+{{- if .Values.web.secrets.backupManager }}
+apiVersion: v1
+kind: Secret
+metadata: 
+  name: backup-manager-secret-{{ .Values.deployEnv }}
+  labels:
+    deployEnv: "{{ .Values.deployEnv }}"
+type: Opaque
+data:
+  BackupManagerKey.json: {{ .Values.web.secrets.backupManager.key | b64enc }}
+  gcloud-service-accnt.json: {{ .Values.web.secrets.backupManager.serviceAccount | b64enc }}
+{{- end }}
+---
+{{- if or .Values.web.secrets.logging.prod .Values.web.secrets.logging.dev }}
```

Comment:
why would this be environmentally keyed?  Every env would have its own helm deployment, describing web.secrets.logging for that environment